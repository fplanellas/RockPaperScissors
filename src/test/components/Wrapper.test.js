import React from 'react';
import { shallow } from 'enzyme';
import Wrapper from '../../components/Wrapper';

describe('testing in component Wrapper', () => {

    test('should be displayed correctly', () => {
        const wrapper = shallow( <Wrapper /> );
        expect( wrapper ).toMatchSnapshot();
    })
      
})