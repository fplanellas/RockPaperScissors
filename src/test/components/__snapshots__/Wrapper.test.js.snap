// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing in component Wrapper should be displayed correctly 1`] = `<styled.div />`;
